app/models/asset.rb

```diff
@@ -0,0 +1,13 @@
+class Asset < ApplicationRecord
+
+  enum type: [
+         :link,
+         :image,
+         :assessment
+       ]
+
+  belongs_to :version
```

Comment:
yeah, it's an "AssessmentVersion".  

I'm conflicted on what to call this. It's really is the assessment, since it's what all the things link to.  The current Assessment model is really just a uuid and links to organization & license.

Do you think it'd be too weird to rename this to "Assessment" and name the existing Assessment to be "AssessmentOwnership" ?